stix_shifter_utils/utils/helpers.py

```diff
@@ -0,0 +1,43 @@
+def dict_merge(dct, merge_dct, add_keys=True):
```

Comment:
I pretty much used from here, just enhanced with list merge:
https://gist.github.com/angstwad/bf22d1822c38a92ec0a9?permalink_comment_id=2622319#gistcomment-2622319

Reply:
I would put that link in the comment for reference